documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -72,19 +70,21 @@ With this configuration applied, topics retain their original names.
 
 == Topic configuration synchronization
 
-Topic configuration is automatically synchronized between source and target clusters.
-By synchronizing configuration properties, the need for rebalancing is reduced.
-
-== Data integrity
-
-MirrorMaker 2.0 monitors source topics and propagates any configuration changes to remote topics, checking for and creating missing partitions.
-Only MirrorMaker 2.0 can write to remote topics.
```

Comment:
@mimaison - I removed this rather than it possibly confusing matters. I don't know if tehre's anything we should add about constraints on writing to topics within the MM2 framework?

Reply:
Yes that makes sense.

Reply:
MM2 does not prevent any actions on remote topics but in many cases users should avoid producing to remote topics.

Reply:
From my experience ... I think they usually understand that they should not produce into them. But what they usually don't understand is that when the consumed from them, the consumed offsets are not synced back into the original cluster.